index.html

```diff
@@ -0,0 +1,206 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+    integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <div class="after-bdy">
+    <div class="portfolio-class">
+      <nav class="flex white-back">
+        <a id="title" href="#">Logo</a>
+        <div class="flex icon-stack-container">
+          <span id="icon-stack"></span>
+        </div>
+      </nav>
+      <main class="main-headline white-back">
+        <article>
+          <h1>
+            I'm Ahmed<br />
+            Glad to see <br />
+            you!
+          </h1>
+          <p>
+            I'm a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you
+            like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+          <div class="socials">
+            <h6>LET'S CONNECT</h6>
+            <ul class="icons_social__container flex">
+              <li>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-linkedin-in"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-github"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-instagram"></i></a>
+              </li>
+            </ul>
+          </div>
+        </article>
+      </main>
+      <section class="projects">
+        <article class="card flex white-back">
+          <img src="./images/project_1.png" alt="tonic project preview" />
+          <h3>Tonic</h3>
+          <ul class="flex project_details">
+            <li>CANOPY</li>
+            <li>BACKEND DEV</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
```

Comment:
The size of this `p` tag is `15px` as per Figma design. Kindly check out the design again. And match font-sizes of similar elements in the design too.
![image](https://user-images.githubusercontent.com/61361037/219712777-25a4a279-caaf-4317-8b4f-e90791c9bf6e.png)